fares-data-build-tool/create-data
# CFDS-284 fares site password policy

## Description

Change fares site password policy to require a special character, lowercase letter, uppercase letter and number

## Testing instructions

Spin up site locally, using the disableAuth query param. Go to account setting and change password page, try to change password by not meeting password policy outlined above - check that error is thrown on UI
